pontoon/base/static/css/download_selector.css

```diff
@@ -0,0 +1,34 @@
+.download-selector {
+    float: right;
+  }
+  
+  .download-selector .selector {
+    padding-right: 0;
+    padding-top: 0;
+  }
+  
+  .download-selector .selector .fa {
+    float: right;
+    background: #3f4752;
+    border-radius: 2px;
+    box-sizing: border-box;
+    color: #AAA;
+    font-size: 16px;
+    height: 32px;
+    width: 32px;
+    padding-top: 8px;
+    text-align: center;
+  }
+  
+  .download-selector.opened .selector .fa {
+    background: #272a2f;
+    border-radius: 2px 2px 0 0;
+  }
+  
+  .download-selector.opened .menu {
+    display: block;
+    top: 32px;
+    right: 0px;
+    bottom: auto;
+    width: 220px;
+  }
```

Comment:
Add newline at the end of the file.

Dedent rules by two spaces to put them to the beginning of the line (you can also check out other CSS files like `pontoon.css` for consistency).